packages/redux/src/analytics/middlewares/__tests__/setUser.test.ts

```diff
@@ -118,7 +128,13 @@ const mockStore = (initialState, middlewares) => {
   return store;
 };
 
-async function dispatchUserChangingAction(store, actionType, userEntity, meta) {
+async function dispatchUserChangingAction(
+  // eslint-disable-next-line
```

Comment:
You can remove the `// eslint-disable-next-line` comment if you add this type to the store parameter:

```js
  store: ReturnType<typeof mockStore>,
```

Reply:
Can you check again?